apps/public-docsite-v9/src/Concepts/BuildTimeStyles.stories.mdx

```diff
@@ -0,0 +1,140 @@
+import { Meta, Source, Link } from '@storybook/addon-docs';
+import imageFile from '../../public/style-resolution.png';
+
+<Meta title="Concepts/Developer/Build time styles" />
+
+## Optimize runtime cost of styles
+
+Fluent UI uses the open source [Griffel.js](https://griffel.js.org/) css-in-js styling engine.
+Make sure to take a look at the following docs to get started with styling in Fluent UI:
+
+- [Quick start](?path=/story/quick-start)
+- [Styling components](?path=/story/concepts-developer-styling-components--page)
+
+While there is nothing wrong with the associated runtime costs of a css-in-js engine, larger and more complex
+applications might want to optimize for performance. In Fluent UI, the expensive runtime work only
+happens on the first render of the component. This one-time work can be further optimized at build time by
+pre-computing and transforming styles. You can refer to the [last section](#what-is-being-optimized) of this page for more details
+about what the build time optimization does.
+
+## Setup build time style transforms
+
+### Webpack
+
+```sh
+yarn add @griffel/webpack-loader
+```
+
+In your webpack config, simply add the `@griffel/webpack-loader` to the list of rules.
+
+```javascript
+// webpack.config.js
+module: {
+  rules: [
+    {
+      test: /\.(ts|tsx)$/,
+      exclude: /node_modules/,
+      use: {
+        loader: '@griffel/webpack-loader',
+      },
+    },
```

Comment:
I would modify this to show the order of loaders: usually there is already `babel-loader` or `ts-loader`. Applying loaders in the wrong order will cause issues.